### ISSUE TITLE
Any example for API with Authorization?

### ISSUE THREAD

**Issue by Antony-Jia** [MasoniteFramework/masonite#662]
### Is your feature request related to a problem?

Any example for API with Authorization? Mostly example is too old to run.

### Describe the solution you'd like

Can Add example with api or vue example for newest release?

### Would this be a breaking change ?

- [ ] Yes

**Comment by girardinsamuel** [comment]
Hi ! Did you read this page https://docs.masoniteproject.com/features/api in the documentation ? Does it answer your question ?

**Comment by Antony-Jia** [comment]
yeah i'd read it and it worked well. But when i integration Authorization, i can't find an complete example and find no where to insert guard to protect api. Just like when i add User Model Authorizes, I need put sometime like 'table.string("role")' into User migrations?

Mationte is good framework, but the example and tutorial is so less or older.
